stdlib/public/core/StringBridge.swift

```diff
@@ -234,35 +243,143 @@ internal func _bridgeCocoaString(_ cocoaString: _CocoaString) -> _StringGuts {
     //   2) If it's mutable with no associated information, then a copy must
     //      happen; might as well eagerly bridge it in.
     //   3) If it's mutable with associated information, must make the call
-    let immutableCopy
-      = _stdlib_binary_CFStringCreateCopy(cocoaString) as AnyObject
+    let immutableCopy =
+      _stdlib_binary_CFStringCreateCopy(cocoaString) as AnyObject
     
-#if !(arch(i386) || arch(arm))
+    #if !(arch(i386) || arch(arm))
     if _isObjCTaggedPointer(immutableCopy) {
       return _StringGuts(_SmallString(taggedCocoa: immutableCopy))
     }
-#endif
+    #endif
     
     let (fastUTF8, isASCII): (Bool, Bool)
     switch _getCocoaStringPointer(immutableCopy) {
     case .ascii(_): (fastUTF8, isASCII) = (true, true)
     case .utf8(_): (fastUTF8, isASCII) = (true, false)
     default:  (fastUTF8, isASCII) = (false, false)
     }
-    let length = _stdlib_binary_CFStringGetLength(immutableCopy)
     
     return _StringGuts(
       cocoa: immutableCopy,
       providesFastUTF8: fastUTF8,
       isASCII: isASCII,
       length: length)
+}
+
+@_effects(releasenone)
+public //SPI(Foundation)
+func _bridgeConstantCocoaString(
```

Comment:
Overall it would be great to not expose so many new functions as ABI, even if they're private/underscored. May not be an option though.